### ISSUE TITLE
Nullpointer exception (NPE) on invalid value in `.editorconfig` property for enabling/disabling rules

### ISSUE THREAD

**Issue by mfederczuk** [pinterest/ktlint#1806]
If a rule property (e.g.: `ktlint_standard_import-ordering`, ...) in the `.editorconfig` file is any value that is neither `enabled` nor `disabled`, then ktlint crashes due to an uncaught `NullPointerException`, which is very ambiguous and doesn't tell about what the problem is.

Empty values are detected as such, and a more helpful exception is thrown.

## Expected Behavior

No crash + a warning that the `.editorconfig` is misconfigured, or at least a more helpful exception message.

## Observed Behavior

Immediately after executing, ktlint crashes due to the following uncaught exception:

```log
Exception in thread "main" java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke "com.pinterest.ktlint.core.api.editorconfig.RuleExecution.name()" because "ruleSetExecution" is null
	[...]
Caused by: java.lang.NullPointerException: Cannot invoke "com.pinterest.ktlint.core.api.editorconfig.RuleExecution.name()" because "ruleSetExecution" is null
	[...]
```

The full exception stacktrace is in the log:

<details>
<summary><code>ktlint --log-level=debug Main.kt</code></summary>

```log
12:49:59.650 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "baseline" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "checkstyle" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "json" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "format" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "html" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "plain" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "plain-summary" id.
12:49:59.654 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Discovered reporter with "sarif" id.
12:49:59.655 [main] DEBUG com.pinterest.ktlint.internal.KtlintCommandLine - Initializing "plain" reporter with {color=false, color_name=DARK_GRAY, format=false}
12:50:00.234 [pool-1-thread-1] DEBUG com.pinterest.ktlint.core.internal.EditorConfigLoader - Resolving .editorconfig files for /tmp/tmp.Qgqx61wzEq/Main.kt file path:
	ktlint_standard_import-ordering: foobar
12:50:00.234 [pool-1-thread-1] DEBUG com.pinterest.ktlint.core.internal.RuleExecutionContext - Editor config properties for file '/tmp/tmp.Qgqx61wzEq/Main.kt': {ktlint_standard_import-ordering=ktlint_standard_import-ordering = foobar}
12:50:00.241 [pool-1-thread-1] DEBUG com.pinterest.ktlint.core.internal.RuleRunnerSorter - Rules will be executed in order below (unless disabled):
           - standard:annotation, 
           - standard:annotation-spacing, 
           - standard:chain-wrapping, 
           - standard:colon-spacing, 
           - standard:comma-spacing, 
           - standard:comment-spacing, 
           - standard:curly-spacing, 
           - standard:dot-spacing, 
           - standard:double-colon-spacing, 
           - standard:enum-entry-name-case, 
           - standard:filename, 
           - standard:final-newline, 
           - standard:import-ordering, 
           - standard:keyword-spacing, 
           - standard:modifier-order, 
           - standard:multiline-if-else, 
           - standard:no-blank-line-before-rbrace, 
           - standard:no-blank-lines-in-chained-method-calls, 
           - standard:no-consecutive-blank-lines, 
           - standard:no-empty-class-body, 
           - standard:no-empty-first-line-in-method-block, 
           - standard:no-line-break-after-else, 
           - standard:no-line-break-before-assignment, 
           - standard:no-multi-spaces, 
           - standard:no-semi, 
           - standard:no-trailing-spaces, 
           - standard:no-unit-return, 
           - standard:no-unused-imports, 
           - standard:no-wildcard-imports, 
           - standard:op-spacing, 
           - standard:package-name, 
           - standard:parameter-list-wrapping, 
           - standard:paren-spacing, 
           - standard:range-spacing, 
           - standard:spacing-around-angle-brackets, 
           - standard:spacing-between-declarations-with-annotations, 
           - standard:spacing-between-declarations-with-comments, 
           - standard:string-template, 
           - standard:unary-op-spacing, 
           - standard:wrapping, 
           - experimental:class-naming, 
           - experimental:comment-wrapping, 
           - experimental:context-receiver-wrapping, 
           - experimental:discouraged-comment-location, 
           - experimental:fun-keyword-spacing, 
           - experimental:function-naming, 
           - experimental:function-return-type-spacing, 
           - experimental:function-start-of-body-spacing, 
           - experimental:function-type-reference-spacing, 
           - experimental:kdoc-wrapping, 
           - experimental:modifier-list-spacing, 
           - experimental:nullable-type-spacing, 
           - experimental:parameter-list-spacing, 
           - experimental:property-naming, 
           - experimental:spacing-between-function-name-and-opening-parenthesis, 
           - experimental:type-argument-list-spacing, 
           - experimental:type-parameter-list-spacing, 
           - experimental:unnecessary-parentheses-before-trailing-lambda, 
           - standard:trailing-comma-on-call-site, 
           - standard:trailing-comma-on-declaration-site, 
           - experimental:function-signature, 
           - standard:argument-list-wrapping, 
           - standard:indent, 
           - standard:max-line-length, 
           - experimental:block-comment-initial-star-alignment
Exception in thread "main" java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke "com.pinterest.ktlint.core.api.editorconfig.RuleExecution.name()" because "ruleSetExecution" is null
	at java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)
	at com.pinterest.ktlint.internal.KtlintCommandLine.parallel(KtlintCommandLine.kt:684)
	at com.pinterest.ktlint.internal.KtlintCommandLine.parallel$default(KtlintCommandLine.kt:642)
	at com.pinterest.ktlint.internal.KtlintCommandLine.lintFiles(KtlintCommandLine.kt:407)
	at com.pinterest.ktlint.internal.KtlintCommandLine.run(KtlintCommandLine.kt:332)
	at com.pinterest.ktlint.Main.main(Main.kt:33)
Caused by: java.lang.NullPointerException: Cannot invoke "com.pinterest.ktlint.core.api.editorconfig.RuleExecution.name()" because "ruleSetExecution" is null
	at com.pinterest.ktlint.core.internal.VisitorProvider.isRuleSetEnabled(VisitorProvider.kt:128)
	at com.pinterest.ktlint.core.internal.VisitorProvider.isRuleEnabled(VisitorProvider.kt:123)
	at com.pinterest.ktlint.core.internal.VisitorProvider.isEnabled(VisitorProvider.kt:102)
	at com.pinterest.ktlint.core.internal.VisitorProvider.visitor$ktlint_core(VisitorProvider.kt:55)
	at com.pinterest.ktlint.core.KtLintRuleEngine.lint(KtLint.kt:435)
	at com.pinterest.ktlint.core.KtLintRuleEngine.lint(KtLint.kt:392)
	at com.pinterest.ktlint.internal.KtlintCommandLine.process(KtlintCommandLine.kt:480)
	at com.pinterest.ktlint.internal.KtlintCommandLine.access$process(KtlintCommandLine.kt:57)
	at com.pinterest.ktlint.internal.KtlintCommandLine$lintFiles$3.invoke$lambda$0(KtlintCommandLine.kt:400)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

## Steps to Reproduce

1. Create a `.editorconfig` file with a rule property with a value that is neither `enabled` nor `disabled`. For example:

   ```editorconfig
   ktlint_standard_import-ordering = disable # note the missing 'd', which is the mistake I originally made
   ```

   or:

   ```editorconfig
   ktlint_standard_import-ordering = foobar
   ```

2. Create a Kotlin file, for example `Main.kt`.
   The contents don't matter.

3. Execute ktlint

   ```console
   $ ktlint Main.kt
   ```

4. See crash

## Your Environment

* Version of ktlint used: `0.48.2`
* Used the command line
* Operating System and version: Fedora Linux 37

**Comment by paul-dingemans** [comment]
Tnx for this report. This is indeed inconvenient and unhelpful message.
